src/cmds/scripts/win_postinstall.py

```diff
@@ -311,9 +345,16 @@ def __svc_helper(svc, username, password):
     svc_name = os.path.basename(svc).replace('.exe', '').upper()
     cmd = [os.path.join(pbs_bin, 'pbs_account.exe')]
     cmd += ['--reg', svc, '-a', username, '-p', password]
-    if __run_cmd(cmd) > 0:
-        __log_err('Failed to register service %s' % svc_name)
-        sys.exit(9)
+    unregcmd = [os.path.join(pbs_bin, 'pbs_account.exe')]
```

Comment:
a) As explained above, In case of failure while executing first trail, we wanted to do extra responsibility for unreg the services.(Only in the case of negative scenario)
And, 
b)This will help user to while un-installing, Now, Its not mandatory to unreg the services. (This will make user life easy). There are many occurrences, user missed to unreg while uninstalling and it will fail to execute win_postinstall script.